core/src/jsMain/kotlin/ScannerBuilder.kt

```diff
@@ -1,20 +1,21 @@
 package com.juul.kable
 
-import com.benasher44.uuid.Uuid
 import com.juul.kable.logs.Logging
 import com.juul.kable.logs.LoggingBuilder
 
 public actual class ScannerBuilder {
-    public actual var services: List<Uuid>? = null
+    public actual var filters: List<Filter>? = null
     private var logging: Logging = Logging()
 
     public actual fun logging(init: LoggingBuilder) {
         logging = Logging().apply(init)
     }
 
     internal actual fun build(): Scanner {
-        val filters = services
+        val filters = filters
```

Comment:
Let's rename the local variable to `scannerFilters` to remove confusion.

```suggestion
        val scannerFilters = filters
```

_You'll also need to edit line 23 below._

Reply:
Good call, it shadows the instance `val`.